app/views/incomes/SameEstimatedPayView.scala.html

```diff
@@ -29,7 +29,7 @@
 @import uk.gov.hmrc.tai.util.ViewModelHelper.currentTaxYearRangeHtmlNonBreak
 
 
-@this(mainTemplate: MainTemplate, loginStatus: LoginStatus)
+@this(mainTemplate: MainTempleteNew, loginStatus: LoginStatus)
```

Comment:
Just being picky here but is loginStatus used anywhere?

Reply:
oh well... it wasn't me who wrote it but i'll remove!